superset/connectors/druid/models.py

```diff
@@ -1240,7 +1268,7 @@ def run_query(  # druid
             qry["limit"] = row_limit
             client.scan(**qry)
         elif (IS_SIP_38 and columns) or (
-            not IS_SIP_38 and len(groupby) == 0 and not having_filters
+            not IS_SIP_38 and (not groupby or len(groupby) == 0) and not having_filters
```

Comment:
I wonder if a simple falsy check here would be more pythonic: `not groupby`? I can't think of an edge case for which it wouldn't give the correct results.

Reply:
@villebro thanks. I agree and made this change elsewhere.